modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -325,4 +323,121 @@ spec:
 [source,terminal]
 ----
 $ oc apply -f my-performance-profile.yaml
-----
\ No newline at end of file
+----
+
+[id="node-tuning-operator-pod-power-saving-config_{context}"]
+== Optional: Power saving configurations
+
+You can enable power savings for a node that has low priority workloads that are co-located with high priority workloads without impacting the latency or throughput of the high priority workloads. Power saving is possible without modifications to the workloads themselves.
+
+[IMPORTANT]
+====
+The feature is supported on Intel Ice Lake and later generations of Intel CPUs. The capabilities of the processor might impact the latency and throughput of the high priority workloads.
+====
+
+When you configure a node with a power saving configuration, you must configure high priority workloads with performance configuration at the pod level, which means that the configuration applies to all the cores used by the pod.
+
+By disabling P-states and C-states at the pod level, you can configure high priority workloads for best performance and lowest latency.
+
+.Power saving configurations
+[cols="1,2", options="header"]
+|====
+|Annotation
+|Description
+
+a|[source,terminal]
+----
+annotations:
+`cpu-c-states.crio.io: "enable"`
+`cpu-freq-governor.crio.io: "<governor>"`
+----
+|Provides the best performance for a pod by disabling C-states and specifying the governor type for CPU scaling. The `performance` governor is recommended for high priority workloads.
+|====
+
+
+.Prerequisites
+
+* You enabled C-states and OS-controlled P-states in the BIOS
+
+.Procedure
+
+. Generate a `PerformanceProfile` with `per-pod-power-management` set to `true`:
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator -v \
+/must-gather:/must-gather:z registry.redhat.io/openshift4/performance-addon-rhel8-operator:v4.11 \
+--mcp-name=worker-cnf --reserved-cpu-count=20 --rt-kernel=true \
+--split-reserved-cpus-across-numa=false --topology-manager-policy=single-numa-node \
+--must-gather-dir-path /must-gather -power-consumption-mode=low-latency \ <1>
+--per-pod-power-management=true > my-performance-profile.yaml
+----
+<1> The `power-consumption-mode` must be `default` or `low-latency` when the `per-pod-power-management` is set to `true`.
+
++
+.Example PerformanceProfile with perPodPowerManagement
```

Comment:
The [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) do allow for using backticks in the title for a block (just not in assembly/module headings):

```
.Example `PerformanceProfile` with `perPodPowerManagement`
```